build-system/tasks/presubmit-checks.js

```diff
@@ -335,6 +337,7 @@ var forbiddenTerms = {
       'extensions/amp-analytics/0.1/cid-impl.js',
       'src/cookies.js',
       'src/experiments.js',
+      'dist.3p/current/integration.js',
```

Comment:
I was getting a presubmit failure due to dist.3p/current/integration.js without these.